src/firecracker/src/api_server_adapter.rs

```diff
@@ -103,6 +103,7 @@ pub fn run_with_api(
     instance_info: InstanceInfo,
     start_time_us: Option<u64>,
     start_time_cpu_us: Option<u64>,
+    boot_timer_present: bool,
```

Comment:
```suggestion
    enable_boot_timer: bool,
```

Reply:
I will rename it **boot_timer_enabled** so that it matches the **api_enabled** in main.rs.